core/storage/user/gae_models.py

```diff
@@ -67,6 +67,17 @@ def get_by_normalized_username(cls, normalized_username):
             cls.normalized_username == normalized_username).get()
 
 
+class UserContributionsModel(base_models.BaseModel):
+    """Tracks statistics for a particular user.
+
+    Instances of this class are keyed by the user id.
+    """
+    # IDs of explorations that this user has created.
```

Comment:
Maybe add a note that these include subsequently-deleted explorations and explorations that are private.